requirements.txt

```diff
@@ -1,5 +1,5 @@
 fastapi[all]==0.92.0
-fastapi-users[all]==10.4.0
+fastapi-users[all]==10.3.0
```

Comment:
I'm curious, why are we staying in `10.3.0`?

Reply:
Because going to 10.4.0 breaks the library internals :D they somehow messed up parameters for `oauth` callback and our authorization does not work there